pkgs/development/python-modules/staticjinja/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, docopt
+, easywatch
+, jinja2
+}:
+
+buildPythonPackage rec {
+  pname = "staticjinja";
+  version = "0.3.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1mxv7yy35657mfxx9xhbzihh10m5lb29fmscfh9q455zd4ikr032";
+  };
+
+  propagatedBuildInputs = [ jinja2 docopt easywatch ];
+
+  meta = with lib; {
+    description = "A library and cli tool that makes it easy to build static sites using Jinja2";
```

Comment:
Would there ever be a case where people would only be interested in the cli tool exclusively?

Reply:
I think that's the majority of people, since using it as a lib is the "advanced mode"

Reply:
But hat's the case for other python packages too. For example I have `pythonPackages.mpsyt` in my system packages